frontend/awx/administration/credential-types/CredentialTypePage/CredentialTypeCredentials.cy.tsx

```diff
@@ -0,0 +1,29 @@
+import { CredentialTypeCredentials } from './CredentialTypeCredentials';
+
+describe('CredentialTypeCredentials', () => {
+  beforeEach(() => {
+    cy.intercept(
+      { method: 'GET', url: '/api/v2/credential_types/*' },
+      { fixture: 'credentialType' }
+    );
+  });
+
+  it('fetches credentials for the correct credential type', () => {
+    cy.intercept(
+      {
+        method: 'GET',
+        url: '/api/v2/credential_types/*/credentials/*',
```

Comment:
```suggestion
        url: '/api/v2/credential_types/1/credentials/*',
```

I don't think we need to include `path` and `initialEntries` on mount if you include the id here. 

```
cy.mount(<CredentialTypeCredentials />);
```